openshift_scalability/ci/scripts/cnv/scale_build_test.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+
+####################################################################################
+#  This is script to run the scale testing  using cluster loader on OCP lab.       #
+#                                                                                  #
+#   - Set VMs quantity                                                             #
+#   - Select scale test by name                                                    #
+####################################################################################
+
+SETUP_PBENCH=$1
+CONTAINERIZED=$2
+CLEAR_RESULTS=$3
+MOVE_RESULTS=$4
+TOOLING_INVENTORY=$5
+TEST_NAME=$6
+VMS_QUENTITY=$7
+CURRENT_TEST=""
+CURRENT_TEST_SCRIPT=""
+CURRENT_TEST_NS=""
+
+# TODO: Set namespace with loop according to number of vms
+if [[ "${TEST_NAME}" == "cirros_test" ]]; then
+    CURRENT_TEST='cnv_vm_cirros'
+    declare -a CURRENT_TEST_NS=('cvn-vm-testing-cirros0' 'cvn-vm-testing-cirros1' 'cvn-vm-testing-cirros2' 'cvn-vm-testing-cirros3')
+elif [[ "${TEST_NAME}" == "fedora_test" ]]; then
+    CURRENT_TEST='cnv_vm_fedora'
+    declare -a CURRENT_TEST_NS=('cvn-vm-testing-fedora0' 'cvn-vm-testing-fedora1' 'cvn-vm-testing-fedora2' 'cvn-vm-testing-fedora3')
+else
+   echo "Wrong test name"
```

Comment:
Can you add some extra information what parameters user should use?

Reply:
it's just 'nice to have' - you can ignore it.
`echo "$TEST_NAME is not a valid option, available options: cirros_test, fedora_test"`

Reply:
DONE